convert/tsdb.go

```diff
@@ -174,6 +102,7 @@ func (rr *indexRowReader) ReadRows(buf []parquet.Row) (int, error) {
 		}
 	}()
 
+	labelsSchema := schema.LabelsProjection(rr.schema)
```

Comment:
are the changes in this method really needed? they feel not related to the sharding